### ISSUE TITLE
Why are your settings not in accordance with the standards of ssl, UDA and SSDA?

### ISSUE THREAD

**Issue by ljjcoder** [google-research/adamatch#3]
Thank you for your excellent work. I'm curious why you didn't keep the same settings as before to experiment? For example, in the SSL field, the commonly used data sets are cifar10 and cifar100, as well as SVHN and STL-10. The commonly used in UDA are office-home and office31. The commonly used in SSDA are Domainnet and Officehome. If you change the settings, it is difficult to compare fairly with previous work.

**Comment by shreejalt** [comment]
@ljjcoder I tried on the officehome dataset with the same hyperparameters on 64x64 using the pytorch dassl repo for adamatch.
I was not able to get good accuracy. Did you tried any?

**Comment by jingzhengli** [comment]
> @ljjcoder I tried on the officehome dataset with the same hyperparameters on 64x64 using the pytorch dassl repo for adamatch. I was not able to get good accuracy. Did you tried any?

Hi, I think that in an SSL setup unlike UDA, the target data cannot be used as a test set even though they have the same distribution.
